test/helpers.py

```diff
@@ -6,16 +6,16 @@
 from typing import Optional
 
 
-def deploy_stub_externally(
+def deploy_app_externally(
     servicer,
     file_or_module: str,
-    stub_variable: Optional[str] = None,
+    app_variable: Optional[str] = None,
     deployment_name="Deployment",
     cwd=None,
     env={},
     capture_output=True,
 ) -> Optional[str]:
-    # deploys a stub from another interpreter to prevent leaking state from client into a container process (apart from what goes through the servicer)
+    # deploys a app from another interpreter to prevent leaking state from client into a container process (apart from what goes through the servicer)
```

Comment:
"a app" you say? ;)

Reply:
(More seriously we should probably watch out for this when we update the user-facing docs)

Reply:
oops this was a mistake, but it's only in tests

Reply:
yep not an issue in a test comment but good to think about (if you haven't been) for user-facing text